recipes/conda-env/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "2.4.5" %}
+
+package:
+  name: conda-env
+  version: {{ version }}
+
+source:
+  fn: conda-env-{{ version }}.tar.gz
+  url: https://github.com/conda/conda-env/archive/v{{ version }}.tar.gz
+  sha256: 5ce2dd1b023a84ee76d19d508e519c1ca84010a60744976aa3caa42ba7ab38f9
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - conda_env
+
+  commands:
+    - which conda-env  # [unix]
+    - where conda-env  # [win]
```

Comment:
I believe I had trouble running these command. Maybe it tries to use the `root` copy of `conda-env`. Hopefully fixing the links helps. 😕